app/models/api/members_api/client.rb

```diff
@@ -23,6 +23,13 @@ def constituency(search_term)
     end
   end
 
+  def member_for_constituency(constituency)
+    member_id = member_id(constituency).object
+    member_name = member_name(member_id).object
+    contact_details = member_contact_details(member_id).object.find { |details| details.type_id == 1 }
+    {name: member_name.name_display_as, email: contact_details.email, address_line1: contact_details.line1, address_line2: contact_details.line2, address_line3: contact_details.line3, address_postcode: contact_details.postcode}
```

Comment:
We need to include lines 4 & 5 in the address as I have seen entries from the API where the address is spread over all 5 lines. Yes even when it's all the same address 😆